### ISSUE TITLE
Feature request: a way to check what versions of each tool are installed

### ISSUE THREAD

**Issue by cb372** [coursier/coursier#1878]
I ran `cs update`, which upgraded a bunch of things. Then I ran `cs list`, hoping it would tell me what versions they had been upgraded to. But `cs list` only prints the names of the tools, not their versions.

(Specifically, I'm trying to work out whether cs has installed the newly released sbt 1.4.0 or not.)

How about a `cs list -v` option to print more detailed information about each tool?

**Comment by alexarchambault** [comment]
I agree this feature would be welcome. `cs list` could be extended to print those versions (when passing `--versions` say).
